packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx

```diff
@@ -0,0 +1,312 @@
+import React from 'react';
+import { CreateDaoSteps, accordionItems, stepsNames } from './Shared';
+import { useCreateDao, ICreateDaoCollaterals } from './CreateDaoContextProvider';
+import {
+  useLayout,
+  Grid,
+  GridItem,
+  Accordion,
+  TextInput,
+  Switch,
+  Box,
+  Button,
+  StyledText,
+  Steps,
+  Split,
+  Info,
+  IconMinus,
+  IconPlus,
+  IconArrowLeft,
+  SPACING,
+} from '@aragon/ui';
+
+const CreateDaoCollateral: React.FC<{
+  setActiveStep: React.Dispatch<React.SetStateAction<CreateDaoSteps>>;
+}> = ({ setActiveStep }) => {
+  const { layoutName } = useLayout();
+  const spacing = SPACING[layoutName];
+  const { collaterals, setCollaterals } = useCreateDao();
+  const {
+    scheduellAddress,
+    scheduellAmount,
+    isScheduellNewDaoToken,
+    challengeAddress,
+    challengeAmount,
+    isChallengeNewDaoToken,
+    isAnyAddress,
+    executionAddressList,
+  } = collaterals;
+
+  const updateCollaterals = (
+    indexType:
+      | 'scheduellAddress'
+      | 'scheduellAmount'
+      | 'isScheduellNewDaoToken'
```

Comment:
typo for scheduleAddress, scheduleAmount, isScheduleNewDaoToken.. and would like to see a type defined for indexType